### DISCUSSION TITLE
Recursive map view

### DISCUSSION THREAD

**Discussion by daniesteban** [bpatrik/pigallery2#445] [Q&A] [answered]
Is there any way to configure pygallery2 to show all the image collection in the map. I mean, If I have the collection divide in folders, one folder by country, when I open pygallery webpage I see a folder list with one example picture, but I can't access to the map until I enter in one of the country folders.
I want to be able to enter the map from the main page to be able to see all the images pinned in a world map.

Thank you. Great job with pigallery.

**Comment by bpatrik** [comment]
You should search for that folder name within `()` and not within `""` to list all photos in that folder and subfolders.

**Reply by daniesteban** [reply]
Thanks for the answer @bpatrik.  I'm not sure I've made myself clear. I'm not doing any search at all, I just want to be able to access to the map when I'm in my pigallery homepage, just in the same way that when I enter to a folder, or when there are pictures in the root folder, but the icon map doesn't appear unless I get into a page with pictures.
I just want to have a global view of all the pictures in a map, no matter where they're located into the images folder configured.

**Comment by bpatrik** [comment, marked as answer]
yes, for that you need to search.

Only those photos show up on the map that are also listed, so if you would like to see a photo on the map, you should get a search query that lists that one. For all your photos you can try searching for something like `.`.

**Reply by daniesteban** [reply]
Ok, I understand. To get the map button I can search without any filter to show the whole collection and the button appears. Nice workaround.
Thank you
